packages/forma-36-react-components/src/components/EntityList/EntityList.tsx

```diff
@@ -16,25 +16,28 @@ export type EntityListProps = {
    * An ID used for testing purposes applied as a data attribute (data-test-id)
    */
   testId?: string;
-} & typeof defaultProps;
-
-const defaultProps = {
-  testId: 'cf-ui-entity-list',
-};
-
-export class EntityList extends Component<EntityListProps> {
-  static defaultProps = defaultProps;
+}
 
-  render() {
-    const { className, children, testId, ...otherProps } = this.props;
+export const EntityList = forwardRef<HTMLUListElement, EntityListProps>(
+  (props: EntityListProps, ref) => {
+    const { className, children, testId, ...otherProps } = props;
     const classNames = cn(styles.EntityList, className);
 
     return (
-      <ul {...otherProps} className={classNames} data-test-id={testId}>
+      <ul
+        ref={ref}
+        className={classNames}
+        data-test-id={testId}
+        {...otherProps}
+      >
         {children}
       </ul>
     );
-  }
-}
+  },
+);
+EntityList.displayName = 'EntityList';
```

Comment:
do we need this? react [mentions](https://reactjs.org/docs/react-component.html#displayname) it's usually inferred from the class name.

Reply:
It's not a class anymore and because of the forwardRef eslint can't infer this. This is what I found about it in [eslint-plugin-rect](https://github.com/yannickcr/eslint-plugin-react/issues/2269)

And this is what I get if I remove `displayName`

![Screenshot 2020-08-24 at 13 19 48](https://user-images.githubusercontent.com/6597467/91039311-a4606c00-e60c-11ea-94a2-518c7911bc7e.png)

Reply:
Ah, I see, `forwardRef` strips the class/function name. Cool, thanks for explaining.

Reply:
Genau! No problem, thanks for the review